test/controllers/roles_controller_test.rb

```diff
@@ -0,0 +1,8 @@
+require "test_helper"
+
+class RolesControllerTest < ActionDispatch::IntegrationTest
+  test "should get index" do
```

Comment:
This test is still failing because we don't have testing infrastructure set up. I recommend deleting the test

Reply:
Thanks for pointing that out.
Done.